central/networkbaseline/service/service_impl.go

```diff
@@ -161,11 +162,24 @@ func (s *serviceImpl) getBaselinePeerByEntityID(
 
 	peers := baseline.GetPeers()
 	for _, peer := range peers {
+		peerType := peer.GetEntity().GetInfo().GetType()
+		peerId := peer.GetEntity().GetInfo().GetId()
 		key := networkgraph.Entity{
-			Type: peer.GetEntity().GetInfo().GetType(),
-			ID:   peer.GetEntity().GetInfo().GetId(),
+			Type: peerType,
+			ID:   peerId,
 		}
 		result[key] = peer
+		// Insert second peer entry for the masked deployment ID
+		// that can be issued by the network graph deployment masker
```

Comment:
This was a tricky issue to investigate and I remember Luis spending some time on it. Could we maybe add some additional info to the comment to explain why are we doing this deterministic masking?